product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_clone_to_template.yaml

```diff
@@ -0,0 +1,332 @@
+          :min_length:
+          :max_length: 100
+        :vm_prefix:
+          :description: VM Name Prefix/Suffix
+          :required_method: :validate_vm_name
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :src_vm_id:
+          :values_from:
+            :options:
+              :tag_filters: []
+            :method: :allowed_templates
+          :description: Name
+          :required: true
+          :notes:
+          :display: :edit
+          :data_type: :integer
+          :notes_display: :show
+        :provision_type:
+          :values_from:
+            :method: :allowed_provision_types
+          :description: Provision Type
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :linked_clone:
+          :values:
+            false: 0
+            true: 1
+          :description: Linked Clone
+          :required: false
+          :display: :edit
+          :default: false
+          :data_type: :boolean
+        :seal:
+          :description: Seal Template
+          :notes: (Applicable for non-Windows OS only)
+          :validation_method: :validate_seal_template
+          :values:
+            false: 0
+            true: 1
+          :required: false
+          :display: :edit
+          :default: false
+          :data_type: :boolean
+        :vm_name:
+          :description: VM Name
+          :required_method: :validate_vm_name
+          :required: true
+          :notes:
+          :display: :edit
+          :data_type: :string
+          :notes_display: :show
+          :min_length:
+          :max_length:
+        :pxe_image_id:
+          :values_from:
+            :method: :allowed_images
+          :auto_select_single: false
+          :description: Image
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :iso_image_id:
+          :values_from:
+            :method: :allowed_iso_images
+          :auto_select_single: false
+          :description: Image
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :pxe_server_id:
+          :values_from:
+            :method: :allowed_pxe_servers
+          :auto_select_single: false
+          :description: Server
+          :required: false
+          :display: :hide
+          :data_type: :integer
+      :display: :show
+    :schedule:
+      :description: Schedule
+      :fields:
+        :schedule_type:
+          :values:
+            schedule: Schedule
+            immediately: Immediately on Approval
+          :description: When to Provision
+          :required: false
+          :display: :edit
+          :default: immediately
+          :data_type: :string
+        :vm_auto_start:
+          :values:
+            false: 0
+            true: 1
+          :description: Power on virtual machines after creation
+          :required: false
+          :display: :edit
```

Comment:
Seems like this field should not be displayed, suggest using `:display: :hide`

Reply:
correct. it shouldn't be displayed. Seems that it is hidden in the attached screenshots regardless of the value of :display.
Nevertheless, I'll change its value to :hide